controllers/pokemon_controller.go

```diff
@@ -0,0 +1,19 @@
+package controllers
+
+import (
+	"fmt"
+	"net/http"
+	"strconv"
+
+	"github.com/rromodev/academy-go-q32021/queries"
+
+	"github.com/gin-gonic/gin"
+)
+
+func GetPokemon(c *gin.Context) {
+	idp := c.Param("id")
+	id, _ := strconv.Atoi(idp)
+	fmt.Println(id)
```

Comment:
Is this print still necessary? It's ok to log stuff throughout the endpoint, if you still want to keep it you could be more descriptive about what this represent, it would be helpful inside tons of logs, like `log.Println("req id: ", id)`